NineChronicles.DataProvider.Executable/Commands/UserStakingMigration.cs

```diff
@@ -1,3 +1,5 @@
+using Nekoyume.Model.Stake;
+
 namespace NineChronicles.DataProvider.Executable.Commands
 {
     using System;
```

Comment:
(Maybe) It should be placed inner the namespace. If @area363 is okay for this and the build passed, it can be merged.

Reply:
@boscohyun Could you place this within the namespace? Otherwise, it looks good to go! 🙇